core/controllers/admin.py

```diff
@@ -204,6 +210,32 @@ def _reload_collection(self, collection_id):
         else:
             raise Exception('Cannot reload a collection in production.')
 
+    def _generate_dummy_exploration(self, dummy_exp_count, dummy_exp_publish):
+        if feconf.DEV_MODE:
+            logging.info(
+                '[ADMIN] %s generated %s number of dummy explorations' %
+                (self.user_id, dummy_exp_count))
+            possible_titles = ['Hulk Neuroscience', 'Quantum Starks',
+                               'Wonder Anatomy',
+                               'Elvish, language of "Lord of the Rings',
+                               'The Science of Superheroes']
+            dummy_exp_list = []
+            for _ in range(int(dummy_exp_count)):
+                title = random.choice(possible_titles)
+                category = random.choice(feconf.SEARCH_DROPDOWN_CATEGORIES)
+                new_exploration_id = exp_services.get_new_exploration_id()
+                exploration = exp_domain.Exploration.create_default_exploration(
+                    new_exploration_id, title=title, category=category,
+                    objective='Dummy Objective')
+                exp_services.save_new_exploration(self.user_id, exploration)
+                dummy_exp_list.append(new_exploration_id)
+                if len(dummy_exp_list) <= int(dummy_exp_publish):
```

Comment:
You could just use the for loop counter var (which is currently _), instead of calculating the length of the list each time?